viewing_party/party.py

```diff
@@ -0,0 +1,90 @@
+# list of disctionaries, where the inner dictionaries represent movies, and have keys: title, genre, and rating 
+movies =[]
+
+def create_movie(title, genre, rating):
+    if not title or not genre or not rating:
+        return None
+    return {"title": title, "genre": genre, "rating": rating}
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data 
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    for movie in user_data['watchlist']:
+        if movie['title'] == title:   
+            user_data["watched"].append(movie)
+            user_data["watchlist"] = [movie for movie in user_data["watchlist"] if not (movie['title'] == title)]
```

Comment:
First time seeing this use instead of `user_data["watchlist"].remove(movie)`

Reply:
Was thinking I should avoid deleting an object in the watchlist while I was iterating through it.